### ISSUE TITLE
'High Contrast Black' theme Calendar WCAG AAA contrast compliance

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1003]
**Describe the bug**
The Calendar component's active day color contrast ratio is not 7:1 or above.

> What version of the Fundamental Library Styles are you using?
v0.9.1-rc.0

> What front-end framework are you implementing in, e.g., Angular, React, etc.?
React

**To Reproduce**
1. Go to https://deploy-preview-928--fundamental-styles.netlify.app/components/calendar.html
2. Check the .is-active Calendar elements state modifiers' color contrast

**Expected behavior**
If we want WCAG AAA compliance, this contrast ratio should be 7:1 or above.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81743158-1b0a9c80-9456-11ea-99c0-45bc41563efc.png)
![image](https://user-images.githubusercontent.com/43764832/81743209-2d84d600-9456-11ea-91ce-d899d92da823.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)
